apps/site/lib/site_web/router.ex

```diff
@@ -217,6 +217,7 @@ defmodule SiteWeb.Router do
     get("/stop/:id", StopController, :get)
     get("/map-config", MapConfigController, :get)
     get("/routes/by-stop/:stop_id", RouteController, :get_by_stop_id)
+    get("/fares/one-way/by-stop/:stop_id", FareController, :one_way_by_stop_id)
```

Comment:
not-pick (non-blocking):  I think it would make sense for this endpoint to be `/stop/:stop_id/fares/on-way` instead of using the `by-stop` path value.  An equally REST-ful design would be something like `/fares/one-way?stop_id=123`.  They'd both return the same data just 2 different approaches (that could coexist)

Reply:
Oh I like that, lemme see what I can do